_includes/page.hbs

```diff
@@ -58,7 +58,7 @@ scripts: []
       </a>
       <div>
         <h1><a href="/">lipu tenpo</a></h1>
-        <div class="subtitle"><a href="/">li kulupu pi toki pona</a></div>
+        <div class="subtitle"><a href="/sona#faq">li alasa e jan pali sin</a></div>
```

Comment:
I wouldn't expect this to draw a lot of attention because its appearance is muted and it doesn't look like a link. If we want to bring more people to the recruitment section I would suggest a menu link.

Reply:
I agree. I made it as more of an easter egg than actually functional heheh (and since I thought something that looked like a link would look ugly in the logo + subtitle)

I think it is fine for now, and if we want to draw more attention to it in future we can do that